### PR TITLE
Fix autocomplete

### PR DESCRIPTION
**Pull request by haslinghuis**
Fix `savedMouseoverItemHandler` which intermittent prevents cli auto complete to function:

```
CliAutoComplete.js:294 Uncaught TypeError: Cannot read properties of undefined (reading '0')
    at HTMLTextAreaElement.<anonymous> (CliAutoComplete.js:294:100)
    at HTMLTextAreaElement.dispatch (jquery.min.js:2:43090)
    at HTMLTextAreaElement.v.handle (jquery.min.js:2:41074)
    at Object.trigger (jquery.min.js:2:71513)
    at HTMLTextAreaElement.<anonymous> (jquery.min.js:2:72108)
    at Function.each (jquery.min.js:2:2976)
    at S.fn.init.each (jquery.min.js:2:1454)
    at S.fn.init.trigger (jquery.min.js:2:72084)
    at b.fire (jquery.textcomplete.min.js:2:3629)
    at b.activate (jquery.textcomplete.min.js:2:7021)
    at jquery.textcomplete.min.js:2:4219
    at searcher (CliAutoComplete.js:237:9)
    at Object.search (CliAutoComplete.js:351:17)
    at b.<anonymous> (jquery.textcomplete.min.js:2:4167)
    at b._search (jquery.textcomplete.min.js:2:2071)
    at b.trigger (jquery.textcomplete.min.js:2:3487)
```

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Pull request merged**
